src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/international/standing.orders/consents/legacy/LegacyGetInternationalStandingOrderConsentsTest.kt

```diff
@@ -0,0 +1,286 @@
+            tppResource.tpp,
+            OBVersion.v3_1_3,
+            signedPayloadConsent
+        )
+
+        assertThat(consent).isNotNull()
+        assertThat(consent.data).isNotNull()
+        assertThat(consent.data.consentId).isNotEmpty()
+        Assertions.assertThat(consent.data.status.toString()).`is`(Status.consentCondition)
+        assertThat(consent.risk).isNotNull()
+
+        // When
+        val result = PaymentRS().getConsent<OBWriteInternationalStandingOrderConsentResponse5>(
+            PaymentFactory.urlWithConsentId(
+                payment3_1_3.Links.links.GetInternationalStandingOrderConsent,
+                consent.data.consentId
+            ),
+            tppResource.tpp,
+            OBVersion.v3_1_3
+        )
+
+        // Then
+        assertThat(result).isNotNull()
+        assertThat(result.data).isNotNull()
+        assertThat(result.risk).isNotNull()
+        assertThat(result.data).isEqualTo(consent.data)
+        assertThat(result.risk).isEqualTo(consent.risk)
+    }
+
+    @EnabledIfVersion(
+        type = "payments",
+        apiVersion = "v3.1.2",
+        operations = ["CreateInternationalStandingOrderConsent", "GetInternationalStandingOrderConsent"],
+        apis = ["international-standing-order-consents"]
+    )
+    @Test
+    fun shouldGetDomesticStandingOrdersConsents_v3_1_2() {
+        // Given
+        val consentRequest = aValidOBWriteInternationalStandingOrderConsent4()
+
+        val signedPayloadConsent =
+            signPayloadSubmitPayment(
+                defaultMapper.writeValueAsString(consentRequest),
+                tppResource.tpp.signingKey,
+                tppResource.tpp.signingKid,
+                true
+            )
+
+        val consent = PaymentRS().consent<OBWriteInternationalStandingOrderConsentResponse4>(
+            payment3_1_2.Links.links.CreateInternationalStandingOrderConsent,
+            consentRequest,
+            tppResource.tpp,
+            OBVersion.v3_1_2,
+            signedPayloadConsent
+        )
+
+        assertThat(consent).isNotNull()
+        assertThat(consent.data).isNotNull()
+        assertThat(consent.data.consentId).isNotEmpty()
+        Assertions.assertThat(consent.data.status.toString()).`is`(Status.consentCondition)
+        assertThat(consent.risk).isNotNull()
+
+        // When
+        val result = PaymentRS().getConsent<OBWriteInternationalStandingOrderConsentResponse4>(
+            PaymentFactory.urlWithConsentId(
+                payment3_1_2.Links.links.GetInternationalStandingOrderConsent,
+                consent.data.consentId
+            ),
+            tppResource.tpp,
+            OBVersion.v3_1_2
+        )
+
+        // Then
+        assertThat(result).isNotNull()
+        assertThat(result.data).isNotNull()
+        assertThat(result.risk).isNotNull()
+        assertThat(result.data).isEqualTo(consent.data)
+        assertThat(result.risk).isEqualTo(consent.risk)
+    }
+
+    /*@EnabledIfVersion(
```

Comment:
Please delete the commented code.

Reply:
Oops, missed that ! I will remove the comments.